### ISSUE TITLE
Network manager applet showing "No connection" despite being connected

### ISSUE THREAD

**Issue by orschiro** [linuxmint/Cinnamon#7217]
```
 * Cinnamon version 3.6.7pe
 * Distribution - Mint 18.3 64bit
 ```

This seems to only happen with WPA2 Enterprise PEAP.

Anyone else with this problem?

![image](https://user-images.githubusercontent.com/170947/35222179-454f8e9e-ff7d-11e7-9240-d590b627e4d7.png)

**Steps to reproduce**

**Expected behaviour**

**Other information**

**Comment by icarter09** [comment]
@pbLegend are you still experiencing this issue?

**Comment by pbLegend** [comment]
Yes, the problem still persists. It happens after hybernation wakeup on seemingly random occasions. It's not always "no connection". Sometimes it also says "connecting to the wireless network" despite an already established connection.

**Comment by timur-g** [comment]
It's been a while, would be good to retest this.

**Comment by pbLegend** [comment]
Seems to be fixed. Haven't had that problem for a while.
